src/modules/ie-objects/controllers/ie-objects.controller.ts

```diff
@@ -62,21 +67,24 @@ export class IeObjectsController {
 		);
 		const visitorSpaceAccessInfo = getVisitorSpaceAccessInfo(activeVisits.items);
 
+		// Get elastic search result based on given parameters
 		const searchResult = await this.ieObjectsService.findAll(
 			queryDto,
 			'_all',
 			referer,
 			user,
-			visitorSpaceAccessInfo
+			visitorSpaceAccessInfo,
+			organisation
 		);
```

Comment:
This function only needs sector to fetch the info, so we should only pass sector to the function.